src/ApiService/ApiService/ApiService.csproj

```diff
@@ -7,6 +7,7 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="CaseExtensions" Version="1.1.0" />
```

Comment:
let's hope there is no supply chain attack hiding here...

Reply:
removed